File: tests/java/src/main/java/com/escape/tests/cases/Case141AliasShadowCache05.java
package com.escape.tests.cases;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.function.Supplier;

/** Task case 141: alias_shadow_cache_05 false-positive/false-negative stress pattern. */
public class Case141AliasShadowCache05 {
    private static final Map<String, Map<String, String>> RETAINED_CACHE = new HashMap<>();
    private static final List<Map<String, String>> RETAINED_AUDIT = new ArrayList<>();
    private static final List<Supplier<String>> RETAINED_HANDLERS = new ArrayList<>();

    public static String execute(String input) {
        String taskName = "alias_shadow_cache_05";
        String raw = (input == null || input.isEmpty()) ? "sample" : input;
        Map<String, String> payload = new HashMap<>();
        payload.put("task", taskName);
        payload.put("entity", "stress");
        payload.put("stage", "evaluation");
        payload.put("input", raw);
        payload.put("checksum", taskName + ":" + raw.length());
        Map<String, String> alias = payload;
        // ESCAPE: alias retained in class-level cache.
        RETAINED_CACHE.put("case_141", alias);
        return "ok";
    }
}

File: tests/go/cases/case_141_alias_shadow_cache_05.go
package escape_tests

var retainedCase141 = []map[string]string{}

func Case141AliasShadowCache05(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "alias_shadow_cache_05",
		"entity": "stress",
		"stage": "evaluation",
		"input": raw,
		"checksum": "alias_shadow_cache_05:" + raw,
	}
	alias := payload
	// ESCAPE: alias retained in package-level sink.
	retainedCase141 = append(retainedCase141, alias)
	return "ok"
}
